src/memtest.rs

```diff
@@ -545,3 +907,77 @@ where
     let str = String::deserialize(deserializer)?;
     Ok(anyhow!(str))
 }
+
+#[cfg(test)]
+mod test {
+    use {
+        super::{MemtestKind, MemtestOutcome, TestObserver},
+        std::{error::Error, fmt},
+    };
+
+    #[derive(Debug)]
+    struct IterationCounter {
+        expected_iter: Option<u64>,
+        completed_iter: u64,
+    }
+
+    // This is just a place holder, it will never be used.
+    #[derive(Debug)]
+    pub struct IterationError;
```

Comment:
```suggestion
    struct IterationError;
```
Mistake when copy-pastaing